File: Production-SPA/src/app/models/worker.ts

export class Worker {
  id: number;
  name: string;
  surname: string;
  jmbg: number;
  dateOfBirth: Date;
  dateOfEmployment: Date;
  accountNumber: string;
  workplaceNumber: number;
  organizationalUnitNumber: number;

  constructor(p?: any) {
    if (p == null) {
      return;
    }
    this.id = p.id;
    this.name = p.name;
    this.surname = p.surname;
    this.jmbg = p.jmbg;
    this.dateOfBirth = p.dateOfBirth;
    this.dateOfEmployment = p.dateOfEmployment;
    this.accountNumber = p.accountNumber;
    this.workplaceNumber = p.workplaceNumber;
    this.organizationalUnitNumber = p.organizationalUnitNumber;
  }
}

File: Production Back/Production.Domen/Worker.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Production.Domen
{
    public class Worker
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public int JMBG { get; set; }
        public DateTime DateOfBirth { get; set; }
        public DateTime DateOfEmployment { get; set; }
        public string AccountNumber { get; set; }
        public int WorkplaceNumber { get; set; }
        public int OrganizationalUnitNumber { get; set; }

    }
}
